From f02150ae042eea42f610f34c55dde8481535ccb1 Mon Sep 17 00:00:00 2001
From: Keshav Kini <keshav.kini@gmail.com>
Date: Mon, 5 Sep 2016 14:25:58 -0500
Subject: [PATCH] Display Debian BTS URLs when listing bugs
To: foo <foofoofoofoo@mailinator.com>

This commit dumps the URL for the bug on bugs.debian.org alongside the
bug ID when reporting bugs for a package.  This allows a user to easily
find more information about the bug if the summary/title alone is
insufficient.
---
 lib/aptlistbugs/logic.rb | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/lib/aptlistbugs/logic.rb b/lib/aptlistbugs/logic.rb
index 7a416d4..4dfb67d 100644
--- a/lib/aptlistbugs/logic.rb
+++ b/lib/aptlistbugs/logic.rb
@@ -689,7 +689,8 @@ class Viewer
                 bug.id = @bugmap.length.next
                 @bugmap[bug.id.to_s] = bug.bug_number
               end
-              bug_str = " b#{bug.id} - ##{bug.bug_number} - #{bug.desc}"
+              bug_link = "http://bugs.debian.org/#{bug.bug_number}"
+              bug_str = " b#{bug.id} - ##{bug.bug_number} [ #{bug_link} ] - #{bug.desc}"
               # TRANSLATORS: "Found" refers to one singular bug
               bug_str += sprintf(_(" (Found: %s)"), "#{bug.found}") if ( ! bug.found.nil? ) && $DEBUG
               # TRANSLATORS: "Fixed" refers to one singular bug
-- 
2.9.3

